### DISCUSSION TITLE
Downloaded Version Info

### DISCUSSION THREAD

**Discussion by iamLazyCode** [REAndroid/APKEditor#146] [General]
Earlier i Used this `java -jar APK..xxx.jar --version` for version number on latest i can no longer do ...because of added security ?

**Comment by REAndroid** [comment]
This is my mistake `--version` is replaced by `-v` or `-version` ( similarly `--help`). I will add it on next release.
Version and Help flags should stay same forever.

NB: To see the actual flags/options run with: `java -jar APK..xxx.jar -h`

Thank you for reporting

**Reply by iamLazyCode** [reply]
> This is my mistake `--version` is replaced by `-v` or `-version` ( similarly `--help`). I will add it on next release.
> Version and Help flags should stay same forever.
> 
> NB: To see the actual flags/options run with: `java -jar APK..xxx.jar -h`
> 
> Thank you for reporting

My pleasure thankyou for noticing it and fixing in next versions
